cpp/modmesh/transform/fourier.hpp

```diff
@@ -69,11 +66,24 @@ void fft(SimpleArray<T1<T2>> const & in, SimpleArray<T1<T2>> & out)
             }
         }
     }
+}
+
+template <template <typename> class T1, typename T2>
+void ifft(SimpleArray<T1<T2>> const & in, SimpleArray<T1<T2>> & out)
+{
+    auto N = in.size();
```

Comment:
It's bad to use `auto` here.  Use a named type for clarity and readability.  Please correct the other similar `auto`s in this file.

Reply:
I have used `size_t` instead of `auto` in this file.